components/build/tekton-chains/chains-controller-deployment.yaml

```diff
@@ -8,12 +8,9 @@ spec:
     spec:
       containers:
       - name: tekton-chains-controller
-        # Use a nightly-build image because the current latest version, 0.8.0, contains
-        # several bugs that make Chains unusable for our use case. Once version 0.9.0
-        # is published, change this reference to that. Contract team fork may be useful
+        # Contract team fork may be useful
         # in future for testing features not yet upstream:
         # image: ghcr.io/hacbs-contract/chains/controller@sha256...
-        image: gcr.io/tekton-nightly/github.com/tektoncd/chains/cmd/controller@sha256:9b4bef2e9d617dddfc0243b78db2ce8b0f176460bc408bd28e5b31cba79bdece
```

Comment:
Not a blocker, but keeping the url commented out could save some time in future, e.g.:

    # Nightly builds of chains if required:
    #image: gcr.io/tekton-nightly/github.com/tektoncd/chains/cmd/controller@sha256...